### PR TITLE
Ukrainian localization

### PR DESCRIPTION
**Pull request by Sergeyzhol**
Added Ukrainian translation to get proper "In Words" in ERPNEXT

### CODE REVIEW

File: num2words/lang_UK.py [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+>>> ' '.join([str(i) for i in splitby3('1')])
+u'1'
+>>> ' '.join([str(i) for i in splitby3('1123')])
+u'1 123'
+>>> ' '.join([str(i) for i in splitby3('1234567890')])
+u'1 234 567 890'
+
+>>> print(' '.join([n2w(i) for i in range(10)]))
+нуль один два три чотири п'ять шiсть сiмь вiсiм дев'ять
```
Comment (mnatalia) [inline num2words/lang_UK.py]: If you want to make pull request зlease be sure that you use correct Ukrainian variant.

сім

File: num2words/lang_UK.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+>>> print(n2w(1001))
+тисяча один
+>>> print(n2w(2012))
+двi тисячi дванадцять
+
+>>> print(n2w(12519.85))
+дванадцять тисяч п'ятсот дев'ятнадцять кома вiсiмдесят п'ять
+
+>>> print(fill(n2w(1234567890)))
+мiльярд двiстi тридцать чотири мiльйона п'ятсот шiстдесят сiмь тисяч
```
Comment (mnatalia) [inline num2words/lang_UK.py]: тридцять сім

File: num2words/lang_UK.py [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+ZERO = (u'нуль',)
+
+ONES_FEMININE = {
+    1: (u'одна',),
+    2: (u'двi',),
+    3: (u'три',),
+    4: (u'чотири',),
+    5: (u'п\'ять',),
+    6: (u'шiсть',),
+    7: (u'сiмь',),
```
Comment (mnatalia) [inline num2words/lang_UK.py]: сім

File: num2words/lang_UK.py [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+    8: (u'вiсiмь',),
```
Comment (mnatalia) [inline num2words/lang_UK.py]: вісім

**Comment by toofun666** [comment]
Can you direct this development to the main project: savoirfairelinux/num2words? Create a PR there?

**Comment by Sergeyzhol** [comment]
Re: [frappe/num2words] Ukrainian localization (#2)



Hi Tufan,

Yes I could, but it looks like I have no permission to add a branch to savoirfairelinux/num2words.
Maybe I do something wrong. I dont`t use GitHub often.

Thursday, September 7, 2017, 8:06:16 PM, Ви написали:





Can you direct this development to the main project: savoirfairelinux/num2words? Create a PR there?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.





-- 
З повагою,
 Sergey                            mailto:sergeyzhol@gmx.net

**Comment by toofun666** [comment]
Hi,
You can go to the original project and click Fork on the upper right hand
side of the page. In this way you can fork the project to your site. Then,
edit the project, put your files and tests in, test on py 2.7, 3.4, 3.5 and
3.6. The commit the changes to your fork and then push the changes.
Open GitHub and create Pull Request if everyrhing goes OK. Should be fairly
easy.

**Pull request closed without merging**